### PR TITLE
more screenshots added

### PR DESCRIPTION
**Pull request by EParzefall**
and some text polishing/updating.

### CODE REVIEW

File: modules/owncloud_web/pages/web_for_users.adoc [hunk 1]
```diff
@@ -76,6 +80,8 @@ Every space must have at least one manager. Managers of a space have all the rig
 * edit the quota of a space,
 * change the roles of members, including those of other managers. Downgrading a manager to viewer or editor is possible.
 
+image:spaces-menu.png[Spaces Menu,width=60%]]
```
Comment (mmattel) [inline modules/owncloud_web/pages/web_for_users.adoc:83]:
```suggestion
image::spaces-menu.png[Spaces Menu,width=60%]]
```

File: modules/owncloud_web/pages/web_for_users.adoc [hunk 2]
```diff
@@ -88,19 +94,21 @@ TIP: If files in one of the spaces are deleted, they are placed in the trashbin
 
 == Right Sidebar
 
-The right sidebar can be opened via the square icon with the dark or gray sidebar under your avatar symbol in the upper right corner. image:sidebar.png
+The right sidebar can be opened via the square icon with the dark or gray sidebar under your avatar symbol in the upper right corner. image:sidebar.png[]
 
 Here you find details about selected files: name, size, last modification time and with whom they have been shared. From here you can also use the `Actions` menu and reach the `Share` section.
 
-image:file-details[File Details]
+image:file-details.png[File Details]
```
Comment (mmattel) [inline modules/owncloud_web/pages/web_for_users.adoc:101]:
```suggestion
image::file-details.png[File Details]
```

File: modules/owncloud_web/pages/web_for_users.adoc [hunk 3]
```diff
@@ -24,7 +24,9 @@ image:top-navigation.png[]
 
 === Application Switcher
 
-The square in the upper left corner with the nine tiles is your application switcher. There you can switch between `Files` and installed apps.
+The square in the upper left corner with the nine tiles is your application switcher. There you can switch between `Files` and `User management` and installed apps.
+
+image:application-switcher.png[Application Switcher]
```
[the remark this answers is not in the document]
Comment (EParzefall) [reply modules/owncloud_web/pages/web_for_users.adoc:29]: Thanks, good to know!

File: modules/owncloud_web/pages/web_for_users.adoc [hunk 4]
```diff
@@ -88,19 +94,21 @@ TIP: If files in one of the spaces are deleted, they are placed in the trashbin
...
+image:file-details.png[File Details]
 
 === Sharing
 
 If you have been using the standard web interface on ownCloud server previously, the new way of sharing may require some getting used to, but it's even simpler now.
 
 Sharing is either done via the icons to the right of a file or folder or via the `Actions` menu or by opening the right sidebar and clicking `Shares`. In any case, the right sidebar provides you a dialog where you can choose between inviting people registered on the Infinite Scale instance by entering their names or email addresses in the `Invite` field and click btn:[Share] or by sharing via link.
 
-A public link also works for external people not registered on the Infinite Scale server. Below the `Share with people` section, provide a name for the link and set an `Expiration date` and a password before hitting btn:[Share].
+image:sharing.png[Sharing]
```
Comment (mmattel) [inline modules/owncloud_web/pages/web_for_users.adoc:109]:
```suggestion

image::sharing.png[Sharing]
```

**Review by mmattel — APPROVED** [review]

File: modules/owncloud_web/pages/web_for_users.adoc [hunk 5]
```diff
@@ -88,19 +93,22 @@ TIP: If files in one of the spaces are deleted, they are placed in the trashbin
 
 == Right Sidebar
 
-The right sidebar can be opened via the square icon with the dark or gray sidebar under your avatar symbol in the upper right corner. image:sidebar.png
+The right sidebar can be opened via the square icon with the dark or gray sidebar under your avatar symbol in the upper right corner. image:sidebar.png[]
```
Comment (EParzefall) [inline modules/owncloud_web/pages/web_for_users.adoc:96]:
```suggestion
The right sidebar can be opened via the square icon with the dark or gray sidebar under your avatar symbol in the upper right corner: image:sidebar.png[]
```

**Pull request merged**